components/server/ee/src/user/eligibility-service.ts

```diff
@@ -212,9 +212,9 @@ export class EligibilityService {
     }
 
     /**
-     * A user may open private repos if he either:
-     *  - not started his free "priate repo trial" yet
-     *  - is has been no longer than 30 days since he started his "priate repo trial"
+     * A user may open private repos if they either:
```

Comment:
**suggestion**: We recently changed this to allow anyone to use private repos, see https://github.com/gitpod-io/gitpod/pull/5170. We could probably remove `mayOpenPrivateRepo` and some of the functions below. Cc @svenefftinge